browser/src/control/Permission.js

```diff
@@ -13,6 +13,9 @@ L.Map.include({
 	setPermission: function (perm) {
 		var button = $('#mobile-edit-button');
 		button.off('click');
+		button.attr('tabindex', 0);
+		button.attr('role', 'button');
+		button.attr('title', 'Enter editing');
```

Comment:
title is for the tooltip. Which yup is nice to have but we should also add alt attribute

Reply:
alt attribute does not exist for div technically. but should I add it anyway? image of the pencil is actually a div with the background.

Reply:
aha so if it's not a button we should either:
1. Turn that thing into a button (and fix all the styles and things that will brake with that change)
2. Or for now use aria attributes https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA

Reply:
its already ARIA compatible for for button role: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/button_role

Reply:
done